File: app/src/main/java/com/twt/zhihu/zhihudaily/NetService/RetrofitApi.java
package com.twt.zhihu.zhihudaily.NetService;

import com.twt.zhihu.zhihudaily.Model.MainBean;
import com.twt.zhihu.zhihudaily.Model.ContentBean;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface RetrofitApi {

    @GET("stories/latest")
    Call<MainBean> getList();

    @GET("news/before/{date}")
    Call<MainBean> getList(@Path("date") String date);
}

File: app/src/main/java/com/twt/zhihu/zhihudaily/netservice/RetrofitApi.kt
package com.twt.zhihu.zhihudaily.netservice

import com.twt.zhihu.zhihudaily.model.MainBean
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface RetrofitApi {

    @get:GET("stories/latest")
    val list: Call<MainBean>

    @GET("news/before/{date}")
    fun getList(@Path("date") date: String): Call<MainBean>
}
